cvprac/cvp_api.py

```diff
@@ -2619,3 +2619,205 @@ def deploy_device(self, device, container, configlets=None,
         if create_task:
             return self._save_topology_v2([])
         return None
+
+    # pylint: disable=invalid-name
+    # Must be discussed with issue aristanetworks/cvprac #94
+    def get_net_element_info_by_device_id(self, device_id):
+        ''' Return a dict of info about a device in CVP.
+
+            Args:
+                device_id (str): Device Id Key / System MAC.
+
+            Returns:
+                net element data (dict): Dict of info specific to the device
+                    requested or None if the name requested doesn't exist.
+        '''
+        self.log.debug('Attempt to get net element data for %s' % device_id)
+        try:
+            element_info = self.clnt.get('/provisioning/getNetElementInfoById.do?netElementId=%s'
+                                         % qplus(device_id), timeout=self.request_timeout)
+        except CvpApiError as e:
+            # Catch an invalid task_id error and return None
+            if 'errorMessage' in str(e):
+                self.log.debug('Device with id %s could not be found' % device_id)
+                return None
+                # pylint: disable=unreachable
+                raise CvpApiError("get_net_element_info_by_device_id:%s" % e)
+        return element_info
+
+    def get_devices_by_container_id(self, key):
```

Comment:
There is already an existing function get_devices_in_container in cvprac.

Reply:
Tested with success using cv_facts